src/event-helper.js

```diff
@@ -94,21 +93,25 @@ export function listenOnce(element, eventType, listener, opt_capture) {
 
 /**
  * Returns  a promise that will resolve as soon as the specified event has
- * fired on the element. Optionally, opt_timeout can be specified that will
- * reject the promise if the event has not fired by then.
+ * fired on the element.
  * @param {!EventTarget} element
  * @param {string} eventType
  * @param {boolean=} opt_capture
- * @param {number=} opt_timeout
+ * @param {function(!UnlistenDef)=} opt_cancel An optional function that, when
```

Comment:
I suppose this is ok since this is relatively rare. But WDYT about cancellable promises? E.g. we could have a type extension with method `cancel` on the promise?